### PR TITLE
Create open positions page

### PR DESCRIPTION
**Pull request by richardluong**

### CODE REVIEW

File: app/controllers/positions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+class PositionsController < ApplicationController
+
+  # GET /positions
+  def index
+    @positions = Position.all
+  end
+
+  # # GET /positions/1
+  # def show
```
Comment (davidwessman) [inline app/controllers/positions_controller.rb]: Remove unused code

File: app/controllers/positions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      redirect_to positions_url, notice: 'Position was successfully updated.'
+    else
+      render :edit
+    end
+  end
+
+  # DELETE /positions/1
+  def destroy
+    @position = set_position
+    @position.destroy
```
Comment (davidwessman) [inline app/controllers/positions_controller.rb]: Use `position = Position.find(params[:id])` and `destroy!`, then you can remove the `set_position` method.

File: app/helpers/positions_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module PositionsHelper
```
Comment (davidwessman) [inline app/helpers/positions_helper.rb]: Remove

File: spec/models/position_spec.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Position, type: :model do
+  it 'has a valid factory' do
```
Comment (davidwessman) [inline spec/models/position_spec.rb]: Remove this spec, not very useful.

**Comment by richardluong** [comment]
Changes done. :)

**Pull request merged**